cmd/gctcli/commands.go

```diff
@@ -1890,12 +1873,6 @@ func cancelBatchOrders(c *cli.Context) error {
 		return errInvalidAsset
 	}
 
-	if c.IsSet("wallet_address") {
-		walletAddress = c.String("wallet_address")
-	} else {
-		walletAddress = c.Args().Get(5)
-	}
-
 	if c.IsSet("side") {
 		orderSide = c.String("side")
 	} else {
```

Comment:
orderSide = c.Args().Get(6) might need to be changed to orderSide = c.Args().Get(5)

Reply:
Fixed https://github.com/gbjk/gocryptotrader/commit/877ef6ecfc